interface/patient_file/summary/add_edit_issue.php

```diff
@@ -813,6 +813,7 @@ function divclick(cb, divid) {
                                             $disabled = (!AclMain::aclCheckIssue($key, '', ['write', 'addonly'])) ? " disabled" : '';
                                             $str = '<input type="radio" name="form_type" value="%s" onclick="newtype(%s)" %s %s>';
                                             echo vsprintf($str, [attr($index), attr_js($index), $checked, $disabled]);
+                                            echo text($value[1] . " "); /*rm - display issue type name */
```

Comment:
hi @ruthkonyn , fix looks and test well. Would also skip the radio button for when $key == 'medical_device' (ie. only process lines 815 and 816 when $key != 'medical_device') since that issue type is not supported by this toggle mechanism (if toggle on it, it is not correctly transitioned, and I don't think spending the work to support that would be worth the effort)

Reply:
thanks, have done this - apologies the commit message was slightly misstyped! - but i don't think there's an easy way to change the message? if it's important i can fix with a recommit

Reply:
one failing test "Test / PHP 8.3 - Apache - MariaDB 10.6 (pull_request) " which i think i can ignore?

Reply:
I'll re-run it.  That test is flaky sometimes with our CI engine and the way github throttles too many parallel test executions.

Reply:
@bradymiller if this looks fine now with the change, let's bring it in.